README.md

```diff
@@ -1,5 +1,41 @@
 # Ruby_TicTacToe
 
-# Project: TicTacToe
+# Tic Tac Toe OOP (Object Oriented Programming)
 
-Using Ruby OOP (Object Oriented Programming) Approach
+-   Original project specification at [The Odin Project](https://www.theodinproject.com/courses/ruby-programming/lessons/oop)
+-   Tic-Tac-Toe [From Wikipedia](https://en.wikipedia.org/wiki/Tic-tac-toe) for more details about Tic-Tac-Toe.
+
+## Built with:
+
+-   Ruby
+
+## How to run the game:
+
+To play Tic-Tac-Toe:
+
+-   Clone or download this project [from github](https://github.com/noomdalv/Ruby_TicTacToe) using git clone https://github.com/noomdalv/Ruby_TicTacToe.git
+- 	Open a terminal window and go to game folder (cd ../Ruby_TicTacToe/)   
+-   Run the game, using command: bin/main.rb
```

Comment:
I would advice using a better styling for this section, get some inspiration [here](https://github.com/othneildrew/Best-README-Template) , for example for the commands you could use this kind of styling:

```
bin/main.rb
```